crates/revm/src/db/web3db.rs

```diff
@@ -61,16 +62,16 @@ impl Database for Web3DB {
         let (nonce, balance, code) = self.block_on(f);
         // panic on not getting data?
         Ok(Some(AccountInfo::new(
-            U256(
+            U256::from_limbs(
                 balance
-                    .unwrap_or_else(|e| panic!("web3 get balance error:{:?}", e))
+                    .unwrap_or_else(|e| panic!("web3 get balance error:{e:?}"))
                     .0,
             ),
```

Comment:
`.into()` should work here instead of the manual conversion.

Though there can be a bit of dependency hell around `primitive_types` if multiple versions end up in the dep tree.

Reply:
it should work but you're right about the dependency hell. iiuc it's between `rust-web3`, `ruint` and `revm`. i'd leave it as is and fix it when we get rid of `primitive-types` completely in `revm` by adopting the `H160 -> B160, etc.` changes.